app/controllers/external_users/claims_controller.rb

```diff
@@ -448,7 +448,7 @@ def redirect_to_next_step
                   title: '%{action_t} %{type} claim page %{step}'
                 }, claim_tracking_substitutions)
 
-    redirect_to edit_polymorphic_path(@claim, step: @claim.next_step!)
+    redirect_to edit_polymorphic_path(@claim, step: @claim.next_step)
```

Comment:
This is redirecting, so the `@claim` instance doesn't need to be mutated here to have the next step as the action called with assign the `form_step` with the `@claim.next_step`.